corehq/ex-submodules/couchforms/management/commands/purge_forms_and_cases.py

```diff
@@ -68,8 +69,11 @@ def handle(self, *args, **options):
     def iterate_forms_and_collect_case_ids(self):
         print "Iterating Through %s XForms and Collecting Case Ids" % len(self.xform_ids)
         for xform in self.forms_accessor.iter_forms(self.xform_ids):
+            # Get app version by fetching app corresponding to xform build_id since xform.form
+            # does not have updated app version unless form was updated for that version
+            xform.app_version_built_with = self.get_xform_build_version(xform)
```

Comment:
it feels a little funny to me to patching this property onto the form if you can just pass it through to `_form_details` instead